src/app/components/Contact.tsx

```
@@ -0,0 +1,174 @@
+              </a>
+            </div>
+
+            <div className="bg-white p-8 rounded-3xl shadow-2xl">
+              <h3 className="text-2xl font-bold text-gray-900 mb-6">
+                Media Sosial
+              </h3>
+              <div className="space-y-4">
+                <a
+                  href="https://api.whatsapp.com/send?phone=6281288957868&text=Saya%20ingin%20mendapatkan%20informasi%20terkait%20jasa%20sablon%20di%20Mata%20Screenprint"
+                  target="_blank"
+                  className="flex items-center gap-4 p-4 bg-green-50 rounded-lg hover:bg-green-100 transition-colors"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)